draft-ietf-scitt-architecture.md

```diff
@@ -165,29 +170,34 @@ Issuers can register their Signed Statements on one or more Transparency Service
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements with a transparency service is akin to a notarization procedure.
+Transparency services therefore act like notaries that each record (register) a single, linear and irrevocable history of statements that were made.
+How these statements are managed or stored---beside the specified registration procedure---is out-of-scope of this document.
+Earlier, similar approaches for specific classes of artifacts exist, such as Certificate Transparency {{-CT}}.
+The SCITT approach follows a more generic paradigm than previous approaches.
+In this SCITT core architecture, the signed statements registered are opaque to the transparency services that act as a notary.
+This "content-agnostic" approach allows SCITT transparency services to be either integrated in existing solutions or to be an initial part of new emerging systems.
+Extensibility is a vital feature of the SCITT architecture, so that requirements from various applications can be accommodated while always ensuring interoperability with respect to registration procedures and corresponding auditability and accountability.
+For simplicity, the scope of this document is limited to use cases originating from the software supply chain domain, but the specification defined is applicable to any other type of supply chain statements (also referred to as value-add graphs), for example, statements about hardware supply chains.
+
+This document also defines message structures for signed statements and defines a profile for COSE receipts {{-RECEIPTS}}, i.e., countersignatures with metadata).
+These message structures are based on the Concise Binary Object Representation Standard {{-CBOR}} and corresponding signing is facilitated via the CBOR Object Signing and Encryption Standard {{-COSE}}.
+The messages structures are defined using the Concise Data Definition Language {{-CDDL}}.
+In essence, the signatures of signed statements and receipts are based on the COSE_Sign1 specification in {{Section 4.2 of -COSE}}.
+As these messages provide the foundation of any transparency service implementation for global and cross-domain application interoperability, they are based on complementary COSE specifications, mainly {{-RECEIPTS}}.
+Support of COSE_Sign1 and support for extensibility of COSE Header Parameter sets is therefore a prerequisite for implementing the interoperable message layer that comes with this document.
+
+In summary, this specification is in support of relying parties to obtain proof that signed statements where checked for their validity in the past and that check of validity can itself be checked again after the fact via transparency services as defined in this document.
```

Comment:
```suggestion
In summary, this specification supports relying parties obtaining proof that signed statements were recorded and checked for their validity at the time they were registered
The validity can be rechecked after the fact via transparency services as defined in this document.
```

Reply:
+1 on removing the last sentence

Reply:
agreement to remove line 195 in suggested change